src/main/java/com/epam/reportportal/auth/integration/github/GitHubUserReplicator.java

```diff
@@ -54,14 +55,18 @@
 @Component
 public class GitHubUserReplicator extends AbstractUserReplicator {
 
+    private final String githubBaseUrl;
+
     @Autowired
     public GitHubUserReplicator(UserRepository userRepository, ProjectRepository projectRepository, DataStorage dataStorage,
-                                PersonalProjectService personalProjectService) {
+                                PersonalProjectService personalProjectService,
+                                @Value("${rp.auth.github.apiUrl:https://api.github.com}") String githubBaseUrl) {
```

Comment:
Please, put the default value into the .yaml file.

Reply:
updated on develop branch